tools/openshift-ci/verification_tests_ci_entrypoint.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/sh
+
+[[ -z "$BUSHSLICER_TEST_ENVIRONMENT" ]] && { echo "BUSHSLICER_TEST_ENVIRONMENT not set"; exit 1; }
```

Comment:
Can we get this from the KUBECONFIG?

Reply:
This is actually a symbolic name. So we can hardcode it to `origin` probably.

Reply:
test environment here is important - origin would do different things than say online_starter - for example, lack of ssh access. A hardcode here should probably be avoided

Reply:
The other option would be to set the other values (like web_console_url, admin_creds, hosts_type, etc) via env var if we do hardcode the value here

Reply:
@cjryan this is only symbolic name for the env. The actual question is about next line it seems.